deployments/mainnet.json

```
@@ -0,0 +1,6 @@
+{
+  "ElementBridge": "0xaeD181779A8AAbD8Ce996949853FEA442C2CDB47",
+  "CurveStEthBridge": "0x0031130c56162e00A7e9C01eE4147b11cbac8776",
```

```suggestion
  "CurveStEthBridge": "0xe09801da4c74e62fb42dfc8303a1c1bd68073d1a",
```
The address in the PR was an old version. Replace with the newest.